src/retro_data_structures/formats/pak_wii.py

```diff
@@ -38,10 +49,12 @@ def _emitparse_header(code: construct.CodeGen) -> str:
 ConstructResourceHeader._emitparse = _emitparse_header
 
 PAKNoData = Struct(
-    _start=construct.Tell,
+    _start=construct.Tell,      # Should always be 0x00
     _header=PAKHeader,
+    _table_of_contents_start=construct.Tell,        #  Should always be 0x40
```

Comment:
I wonder if it makes sense for these things to have something like
```py
if game == CORRUPTION:
  assert PAKNoData._table_of_contents_start == 0x40
```
Henrique probably knows better there tho

Reply:
You can assert things in Construct, but I don't think there's a need for it. PAKHeader is size 0x40